grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -187,17 +187,17 @@
         });
     }
 
-    final ResponseHeaders defaultResponseHeaders() {
+    public final ResponseHeaders defaultResponseHeaders() {
```

Comment:
```suggestion
    protected final ResponseHeaders defaultResponseHeaders() {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!